include/scalar_node.hh

```diff
@@ -15,6 +15,7 @@
 #include <memory>
 #include "config.hh"
 #include "assert.hh"
+#include "VCL_v2_include.hh"
```

Comment:
VCL should be included only in processor_*.cc before "processor.hh" Otherwise we include VCL implementations with possibly incorrect SIMD size.